agents/dev/stimulate_smtp_metric.py

```diff
@@ -0,0 +1,78 @@
+import logging
+import os
+import sys
+import threading
+
+from ..smtp.smtp import SMTPAgent
+
+# Add the root directory to sys.path
+sys.path.insert(
+    0,
+    os.path.abspath(os.path.join(os.path.dirname(__file__), '..'))
+)
+
+
+# Simulate CPU load
+def start_cpu_stress():
+    for i in range(10):
+        print(i)
+
+
+# Simulate RAM load
+def ram_stress():
+    _ = ['x' * 1024 * 1024 for _ in range(100)]  # Allocate ~100MB temporarily
+
+
+# Simulate disk activity
+def disk_stress(file_path='/tmp/stress_test_file.txt'):
+    with open(file_path, 'w') as f:
+        for _ in range(10000):
+            f.write('stress test line\n')
+
+
+if __name__ == '__main__':
+    agent = SMTPAgent()
+
+    # Create logger
+    logger = logging.getLogger('metrics')
+    logger.setLevel(logging.INFO)
+
+    # Create file handler
+    log_file_path = os.path.join(
+        os.path.dirname(__file__),
+        '../smtp/logs/agent.log'
```

Comment:
The path to the log file is hard-coded: ../smtp/logs/agent.log, but it is better to take it from SMTPAgent.log_dir
log_file_path = os.path.join(agent.log_dir, 'agent.log')